### PR TITLE
Unit2 project

### PR DESCRIPTION
**Pull request by alaurenz**

### CODE REVIEW

File: Unit2/Unit2-Project/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Unit 2 Project: Add an SQL to your full-stack app
```
Comment (ecwalsh) [inline Unit2/Unit2-Project/README.md]: Missing word I think.

File: Unit2/Unit2-Project/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+This project is a follow-on to the Unit1 Project. In this project you will be integrating an SQL database into the full-stack app you built for the Unit 1 project. You will get practice with database design and making a variety of SQL queries in an application context.
+
+
+### SPECIFICATIONS
+
+* Your app must use an SQL-based database system such as MySQL or PostgreSQL. The SQL database will replace Mongo, but we recommend doing this incrementally, so don't worry about fully removing Mongo until you get the SQL database and each of the queries working within your app.
+* Replace each Mongo query with a query to your SQL-based database. You can either use straight SQL or a library such as [Sequelize](https://sequelize.org/).
+* You must have at least 3 tables in your database
+* Ensure each table has a primary key
+* At least one table should have a foreign key constraint to ensure data integrity
```
Comment (ecwalsh) [inline Unit2/Unit2-Project/README.md]: Do you think it should read "foreign key constraint to another table" or is that implicit enough?

**Review by ecwalsh — APPROVED** [review]

File: Unit2/Unit2-Project/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* Add an index to at least one table to improve performance
+* Add functionality to your app to make an SQL query that involves a `JOIN` operation
```
Comment (raevangregson) [inline Unit2/Unit2-Project/README.md]: Add functionality to your app to make **a** SQL query that involves a `JOIN` operation

File: Unit2/Unit2-Project/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+### OVERVIEW & GOALS
+
+This project is a follow-on to the Unit1 Project. In this project you will be integrating an SQL database into the full-stack app you built for the Unit 1 project. You will get practice with database design and making a variety of SQL queries in an application context.
+
+
+### SPECIFICATIONS
+
+* Your app must use an SQL-based database system such as MySQL or PostgreSQL. The SQL database will replace Mongo, but we recommend doing this incrementally, so don't worry about fully removing Mongo until you get the SQL database and each of the queries working within your app.
+* Replace each Mongo query with a query to your SQL-based database. You can either use straight SQL or a library such as [Sequelize](https://sequelize.org/).
```
Comment (raevangregson) [inline Unit2/Unit2-Project/README.md]: I would replace the language of "straight sql" with the term "raw sql"

**Pull request merged**